client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -63,20 +64,34 @@ const SiteSettingsSecurity = ( { rewindActive, site, siteId, siteIsJetpack, tran
 
 	return (
 		<Main className="settings-security__main site-settings">
-			<QueryRewindStatus siteId={ siteId } />
+			<QueryRewindState siteId={ siteId } />
 			<DocumentHead title={ translate( 'Site Settings' ) } />
 			<JetpackDevModeNotice />
 			<SidebarNavigation />
 			<SiteSettingsNavigation site={ site } section="security" />
-			{ rewindActive && <Backups /> }
+			{ ( includes( [ 'active', 'awaitingCredentials', 'provisioning' ], rewindState.state ) ||
```

Comment:
this was clearly an existing thing, but doesn't it feel kind of odd to have this setting area appear and disappear without explanation? it seems like it should probably always be there, either as the credentials form itself or as a nudge to upgrade to a Rewind plan - or in the case for now, always unless `unavailable` perhaps?

just thinking out loud here